src/dataDisplay/personPanel/__test__/personPanel.test.js

```diff
@@ -43,10 +47,15 @@ describe('<PersonPanel />', () => {
     });
 
     it('Should render with the root classes', () => {
+        const rootOverride = 'makeStyles-root-123';
         const wrapper = mount(
             <MockedTheme>
+                cl
```

Comment:
What is `cl` here?  Was this added by mistake?

Reply:
Good catch. It must have been a think I was typing in one window and was typing in the other.

Reply:
Removed.

Reply:
Looks good.  Resolving.